content/realtime/channels.textile

```diff
@@ -205,7 +205,7 @@ bc[swift]. realtime.channels.get("chatroom").attach { error in
   print("'chatroom' exists and is now available globally in every data center")
 }
 
-Clients attach to a channel in order to participate on that channel in any way (either to publish, subscribe, or be present on the channel).
+Clients attach to a channel in order to <span lang="default">participate on that channel in any way (either to publish, subscribe, or be present on the channel)</span><span lang="jsall,ruby,swift">subscribe or be present on a channel. It is still possible to "publish without attaching":#transient-publish to a channel however</span>.
```

Comment:
I fear this is "buggy" in that we'll have to remember to update this as we support transient publishing.  Could transient publishing not be added to our feature matrix, and this text could be generic enough instead to state where transient publishing is supported, an attach will not necessarily subscribe etc.  I think we could then link "See if your chosen SDK supports transient publishing" for example.  Once we upgrade to 1.2, we could probably then remove that completely and assume all support it in full.